### DISCUSSION TITLE
Feature request: support double splat in paths

### DISCUSSION THREAD

**Discussion by criccomini** [duckdb/duckdb#6286] [Ideas]
I would like to be able to do

    SELECT * FROM read_json_auto("/some/root/**/*.json");

And have DuckDB query all JSON files recursively from `/some/root`.

I'd like to do this with S3 as well:

    SELECT * FROM read_json_auto("s3://some-bucket/some/root/**/*.json");

**Comment by Tishj** [comment]
Relevant PR: #6627

**Comment by Mytherin** [comment]
This is implemented on the latest dev build now
